integration/test/test.sh

```diff
@@ -4,5 +4,13 @@ echo "Dir : $SCRIPT_DIR"
 cd $SCRIPT_DIR/test
 tag=${1?Specify valid test tag}
 export PATH=$PATH:/usr/local/go/bin
+if [ ! -n "$VERSION" ]; then
+  echo "VERSION not supplied."
```

Comment:
minor: from a debugging/logging perspective it might be better to move this echo below the next line and change the echo to say, "VERSION not supplied, using $VERSION"